src/shared/components/what-can-you-build-section/accordion/accordion-item/index.js

```diff
@@ -0,0 +1,84 @@
+import React, { Component } from 'react';
```

Comment:
I don't know what has been your approach for the shared directory, but as it seemed strange to me, I decided just to point it out.

I know that your accordion component only exists in the `what-can-you-build-section`, but since it is a generic component I completely agree that it should be in the `shared/components`. However, seems quite strange to me the `shared/components` separation regarding the page/section, where it is used. For instance, if you have a new section in the future that uses this accordion, you will have to change this structure.

Reply:
Totally agree 👍 

Splitting the page into sections makes perfect sense, but putting those under `shared/components` doesn't.

A better solution is to create folders inside ´src/pages/home´ with the various sections, and on the `src/shared/components` folder only goes the components that can be used in multiple sections, hence the folder name.

Reply:
☝️ this can be done in another PR.

Reply:
Ok, I see. I'll fix that in a future PR.